UikitDz/UikitDz/PayVC.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  Pay.swift
+//  UikitDz
+//
+//  Created by Николай Панов on 30.06.2021.
+//
+
+import UIKit
+
+final class PayVC: UIViewController {
+    
+    // MARK: - Public Properties
+    var name = ""
```

Comment:
var name = String() 
```suggestion
    var name = String() 
```

лучте так создавать пустую строку